core/templates/dev/head/pages/admin/admin.html

```diff
@@ -69,6 +69,53 @@
     </div>
 
     {% include 'pages/footer_js_libs.html' %}
+      
+    <!--CollectionEditor.js-->
+    <script src="/templates/dev/head/pages/collection_editor/collection-editor.module.js"></script>
+    <script src="/templates/dev/head/pages/collection_editor/collection-editor.controller.js"></script>
```

Comment:
Do not include script tags, they are replaced with require statements

Reply:
Done.